ansible/roles/nginx/tasks/main.yml

```
@@ -0,0 +1,54 @@
+    name: nginx
+    state: started
+    enabled: yes
+  ignore_errors: "{{ ansible_check_mode }}"
+
+- name: Create log directories
+  file:
+    path: "/var/log/nginx/{{ project_name }}"
+    state: directory
+    owner: www-data
+    group: adm
+    mode: '0755'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)